enclave-runner/src/loader.rs

```diff
@@ -263,6 +263,10 @@ impl<'a> EnclaveBuilder<'a> {
         self
     }
 
+    fn initialized_args_mut(&mut self) -> &mut Vec<Vec<u8>> {
```

Comment:
@mzohreva  We came to this commit after investigating some troubles related to command arguments parsing. It seems the `initialized_args_mut()` would be only called when there are actually arguments being passed (when `arg()` or `args()` is called). So in case no arguments are passed, the `cmd_args` would still remain `None`, `enclave` would not be inserted and the `args` of the enclave process as read by its `main` would be empty. Is this intentional ?

Reply:
I think this is a bug, I should have changed `EnclaveBuilder::build()` to call `initialized_args_mut`. I'll create a tracking issue for it.